arm-apimanagement/2016-07-07/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1234 @@
+          "default": "None",
+          "enum": [
+            "None",
+            "External",
+            "Internal"
+          ],
+          "x-ms-enum": {
+            "name": "VirtualNetworkType",
+            "modelAsString": false
+          }
+        }        
+      },
+      "required": [
+        "publisherEmail",
+        "publisherName"
+      ],
+      "description": "Properties of an API Management service resource description."
+    },
+    "ApiManagementServiceSkuProperties": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Sku.",
+          "externalDocs": {
+            "url": "https://azure.microsoft.com/en-us/pricing/details/api-management/"
+          },
+          "enum": [
+            "Developer",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuType",
+            "modelAsString": false
+          }
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Capacity of the Sku (number of deployed units of the Sku). Default value is 1.",
+          "default": 1
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "API Management service resource Sku properties."
+    },
+    "ApiManagementServiceResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the created API Management service.",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "description": "Datacenter location of the API Management service."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the API Management service.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of the API Management service.",
+          "readOnly": true
+        },
+        "etag": {
+          "type": "string",
+          "description": "ETag of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags of the API Management service."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApiManagementServiceProperties",
+          "description": "Properties of the API Management service."
+        },
+        "sku": {
+          "$ref": "#/definitions/ApiManagementServiceSkuProperties",
+          "description": "Sku properties of the API Management service."                   
+        }
+      },
+      "required": [
+        "sku",
+        "properties",        
+        "location"
+      ],
+      "description": "Description of an API Management service resource."
+    },
+    "ApiManagementServiceUpdateParameters": {
```

Comment:
#Fixed

---
In reply to: [84337976](https://github.com/Azure/azure-rest-api-specs/pull/573#discussion_r84337976) [](ancestors = 84337976)